lib/commands/start.js

```diff
@@ -61,6 +61,8 @@ class StartCommand extends Command {
                 return this.ui.run(processInstance.enable(), 'Enabling instance startup on server boot');
             }).then(() => {
                 this.ui.log(`You can access your blog at ${instance.config.get('url')}`, 'cyan');
+                this.ui.log(`\nGhost uses direct mail by default`, `green`);
```

Comment:
Just as a code style note these don't need to be backticks, should probably be normal single quotes just to stay consistent.